website/docs/run/overview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Overview
+description: Overview
+sidebar_position: 1
+slug: overview
+---
+
+There are two factors to consider when deciding how to run GATK-SV. 
+
+1. **Which platform you would like to use for running GATK-SV?** 
+   You may run GATK-SV on the following platforms. 
+   - [Terra.bio](https://terra.bio): A user-friendly cloud-native platform for scalable data analysis. 
+     The primary focus of this documentation is on supporting the execution of GATK-SV within the Terra platform.
+    
+   - [Cromwell](https://github.com/broadinstitute/cromwell): 
+     You may run GATK-SV on a self-hosted and managed cromwell instance, which is ideal for 
+     power-users and developers. We provide guidelines for this option in the 
+     [_advanced guides_](/docs/advanced/cromwell) section.
+
+2. **How many samples are you planning to process?**
+    We have developed optimized flavors of GATK-SV allowing you to select the one that best suits 
+    your input size for minimal cost and shorter execution time.
+
+    | Execution Mode       | Ideal input size   | Per-sample execution cost | Per-sample execution time |
+    |----------------------|--------------------|---------------------------| ------------------------- |
+    | Singe-sample         | <100 samples       | $7.00-$10.00              | 24-36 hours               | 
+    | Single-batch         | 100-500 samples    | $2.00-$2.50               || 
+    | Multi-batch (cohort) | >200 samples       |                           ||
+
+
+## GATK-SV on Terra
+
+Please refer to the following documentation on running the GATK-SV pipeline within the Terra platform. 
+
+- **Single-sample on Terra**: We have developed a 
+  [Terra workspace](https://app.terra.bio/#workspaces/help-gatk/GATK-Structural-Variants-Single-Sample)
+  for running GATK-SV on a single sample, which contains the related documentation. 
+- [Single-batch on Terra](batch.md);
```

Comment:
I am hesitant about advertising the cohort mode Terra workspace (and single batch, because they are in a single workspace) on the website while it is not publicly available. I think this should at least be noted so that users don't think it's available. 

I'm also not sure that we want to make it seem like single-batch and cohort mode are two entirely different things. There are some differences but currently they share a workspace and more than half the pipeline is identical.

Reply:
That part is mainly from the docs we have on the Terra workspace. I also think that part can benefit from some simplification.

Reply:
Yes, I agree. That part is from the docs we have on Terra workspace. We do not need to keep them in sync, we can improve here and change the workspace docs once these docs are thoroughly improved.